src/core/automount.c

```diff
@@ -732,12 +732,7 @@ static int automount_start_expire(Automount *a) {
 }
 
 static void automount_stop_expire(Automount *a) {
-        assert(a);
```

Comment:
I think we should keep the more visible assertion here. Or, just inline the function since it has only one caller?